src/main/java/at/hannibal2/skyhanni/utils/CollectionUtils.kt

```diff
@@ -313,4 +313,7 @@ object CollectionUtils {
     fun <K, V : Any> Map<K?, V>.filterNotNullKeys(): Map<K, V> {
         return filterKeys { it != null } as Map<K, V>
     }
+
+    fun <K, V> Map<K, V>.containsKey(vararg keys: K) = keys.all { this.keys.contains(it) }
```

Comment:
this easily conflicts with the existing `Map.containsKey` this is also misnamed as its a vararg of keys so it should be `containsKeys` also when changing to `containsKeys` change `this.keys.contains(it)` to `this.containsKey(it)`